bin/flow_number_generator.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by vsazel on 28.2.17.
+ * Generation of randomized flow numbers array.
+ */
+
+/**
+ * Function returns array with flow numbers generated with amount specified as percentages in <b>flowPercentages</b> of <b>totalCount</b>
+ * @param {number} totalCount total count
+ * @param {Array.<number>} flowPercentages array
+ * @param {number=} randGen seed based random generator (LCG)
+ */
+'use strict';
+function generateFlowArray(totalCount, flowPercentages, randGen) {
+  if (!Array.isArray(flowPercentages)) {
+    throw new TypeError('flowPercentages must be an array');
+  }
+  let retArray = [];
+  for (let i = 0; i < flowPercentages.length; i++) {
+    for (let j = 0; j < (Math.ceil((flowPercentages[i] / 100) * totalCount)); j++) {
+      retArray.push(i);
+    }
+  }
+  return shuffle(retArray, randGen);
+}
+
+/**
+ * Function randomly shuffles array (<a href="https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle">Fisher–Yates_shuffle</a>)
+ * @param {Array.<number>} arr array
+ * @param {number=} randGen seed based random generator (LCG)
+ */
+function shuffle(arr, randGen) {
+  let temp, j, i = arr.length;
+  let rand = randGen;
```

Comment:
I think this can be `const` (or even just not have it, and use `randGen`, or rename the `randGen` parameter to the function to `rand`).

This will let us know without having to look through every other line in the function, that it's not being reassigned at any point.